src/main/target/CRAZYBEEF3FR/target.h

```diff
@@ -136,11 +153,10 @@
 #define ADC_INSTANCE            ADC1
 #define CURRENT_METER_SCALE_DEFAULT 2350
```

Comment:
If you are using new scale for the DX, you can put it inside a conditional for the DX.

Reply:
haha , accdording to mikeller's review comments,  i will not change this scale to 1175 ,although 1175 is correct for new version of crazybeef3 fr/fs/dx, instead we will update web page to describe this current meter scale clearly.

Reply:
Fair enough 👍